docs/Intro-Start-Here/HT-Flashing-The-Firmware.md

```diff
@@ -4,36 +4,100 @@ sidebar_position: 4
+Flashing the firmware is easy to do, however it is important to do it correctly to prevent errors or board
+misconfiguration.  A guide to downloading FOME firmware and setting up TunerStudio can be found
+[here](/Intro-Start-Here/Where-To-Get-Firmware/).
 
-## Load the firmware onto the ECU
+## Flashing the firmware onto the ECU with FOME console
 
-Once you have downloaded the [latest firmware bundle](https://github.com/FOME-Tech/fome-fw/releases), extract it then navigate through the *console* folder to *fome_console.exe* and launch it. Ensuring that TS is **not** running, plug your ECU into the computer. The console should detect the board and show a similar screen to below:
+After confirming the firmware to flash to the ECU, software to do so must be used.  FOME provides a utility, the **FOME
+console**, which is capable of flashing firmware to supported ECUs and is the recommended way to flash FOME firmware..
+The FOME console is included with each firmware release, and bundles can be downloaded from [the release
+page](https://github.com/FOME-Tech/fome-fw/releases).  As problems can arise during flashing, other tools and mechanisms
+that are available for this procedure are documented below in [Other firmware flashing
+tools](#other-firmware-flashing-tools).
+
+After extracting the bundle contents, navigate to the `console/` directory and run the FOME console program (e.g.
+`fome_console.exe` if using Windows).  After first confirming **TunerStudio is not running**, connect the ECU to the
+computer.  FOME console will automatically detect the board and present similar to the display below:
 
 ![image](Flashing-FW/fomeconsole.png)
 
-Click *Update Firmware* and leave the board plugged in until the prompt screen indicates that the update was successful. Only once the update has given this message, unplug and re-plug the ECU into the computer to reboot it.
+Next, click *Update Firmware* and leave the board connected until the display notifies that the update was successful.
+Only once the console has presented the success message, disconnected and re-connect the ECU into the computer to reboot
+it with the new firmware.
 
-If this does not complete successfully, the firmware may need to be loaded using STM32 Cube Programmer covered later.
+If the flashing does not complete successfully, the firmware may need to be loaded using another mechanism documented
+below in [Other firmware flashing tools](#other-firmware-flashing-tools).
 
 ## Setting up TunerStudio
 
-Open TS with the ECU plugged in, it should recognise that the ECU definition is out of date and come up with a prompt to download the new definition.
+With the ECU connected, open TunerStudio with the associated project.  TunerStudio will recognise that the project's ECU
+definition is out of date and display a prompt to download or supply the new definition.
```

Comment:
a screenshot here with which button to push would be helpful, it's not entirely clear the first time you see that window

Reply:
I'll add it if you can provide one; I'm not setup for it at the moment.